collectors/o365/local/run-sam.sh

```diff
@@ -7,7 +7,7 @@ SRC_SAM_TEMPLATE="${SCRIPT_DIR}/sam-template.yaml"
 SRC_ENV_FILE="${SCRIPT_DIR}/${ENV_FILE_NAME}"
 SRC_EVENT_FILE="${SCRIPT_DIR}/events/${EVENT_FILE_NAME}"
 RUN_DIR=${SCRIPT_DIR}/../
-
+PROFILE_NAME=""
```

Comment:
let's use `default` profile instead of  `""`.

Reply:
Done